AppKit/CPView.j

```diff
@@ -3511,7 +3703,7 @@ var CPViewAutoresizingMaskKey       = @"CPViewAutoresizingMask",
         [self _decodeThemeObjectsWithCoder:aCoder];
 
         [self setAppearance:[aCoder decodeObjectForKey:CPViewAppearanceKey]];
-
+        
```

Comment:
Open all of the files you edited and save them so the extra whitespace is trimmed.